src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -35,6 +40,223 @@ public ProjectView()
             _versions.ItemContainerStyle = style;
         }
 
+        private TextBox _textBox;
+
+        private TextBox TextBox
+        {
+            get
+            {
+                if (_textBox == null) return _versions.Template.FindName("PART_EditableTextBox", _versions) as TextBox;
+                return _textBox;
+            }
+            set
+            {
+                _textBox = value;
+            }
+        }
+
+        private PackageDetailControlModel _packageDetailControlModel;
+
+        private PackageDetailControlModel PackageDetailControlModel
+        {
+            get
+            {
+                if (_packageDetailControlModel == null) return (PackageDetailControlModel)DataContext;
```

Comment:
I think you mean to set the field here, then return the field? Otherwise, the field is still `null`.
```suggestion
                if (_packageDetailControlModel == null)
                {
                    _packageDetailControlModel = (PackageDetailControlModel)DataContext;
                }
```